docs/index.asciidoc

```diff
@@ -18,3 +18,6 @@ END - GENERATED VARIABLES, DO NOT EDIT!
 
 include::{include_path}/plugin_header.asciidoc[]
 
+==== Description
+
+Reads newline-delimited EDN format data.
```

Comment:
since this codec can also encode Logstash Events to EDN format, I suggest adding "Reads and produces"